includes/Stories_Renderer/Renderer.php

```diff
@@ -550,4 +591,81 @@ protected function get_content_overlay() {
 
 	}
 
+	/**
+	 * Renders the lightbox markup for non-amp pages.
+	 *
+	 * @return void
+	 */
+	protected function render_stories_with_lightbox_noamp() {
+		?>
+		<div class="web-stories-list__lightbox">
+			<amp-story-player width="3.6" height="6" layout="responsive">
```

Comment:
Can you explain these sizes?

Reply:
We use 360x600 as the default size/ratio for amp-story-player.

Using 3.6x6 with `layout="responsive"` should have the exact same result as 360x600. ([see AMP layout examples](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/layouts_demonstrated/))

Although I'd say using `width="360" height="600" seems more readable :-)

Reply:
It's for the ratio for our layout, for which `amp-story-player` should follow the layout.
>https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/?format=websites